gcloud/logging/config_service_v2_api.py

```diff
@@ -14,64 +14,123 @@
 #
 # EDITING INSTRUCTIONS
 # This file was generated from the file
-# https://github.com/google/googleapis/blob/7710ead495227e80a0f06ceb66bdf3238d926f77/google/logging/v2/logging_config.proto,
+# https://github.com/google/googleapis/blob/master/google/logging/v2/logging_config.proto,
```

Comment:
See our previous discussion in #1422.

If you aren't sure how to get the SHA, then I am happy to help you write some code.